lib/project_types/script/messages/messages.rb

```diff
@@ -5,6 +5,9 @@ module Messages
     MESSAGES = {
       script: {
         error: {
+          deprecated_ep: "This project uses Extension point %s which has been deprecated.
+ This Script will no longer function in production.",
```

Comment:
Nit: right now it would display a newline in between the sentences. Usually for multi-sentence messages we keep it all on the same line and let the terminal wrap the text. 

```suggestion
          deprecated_ep: "This project uses Extension point %s which has been deprecated. "\
                         "This Script will no longer function in production.",
```